### DISCUSSION TITLE
Let us know what you think!

### DISCUSSION THREAD

**Discussion by ptaylor9999** [amzn/ads-advanced-tools-docs#5] [Polls]
Hello everyone - we'd like to get your opinion to help us prioritize the tools and samples that our team will publish to this repository. For this first poll, we're interested in the programming language that you primarily use to code against the Amazon Ads API. If you select "other", we'd appreciate if you include a comment to let us know what language you use. Also please feel free to leave any other comments you have!

**Comment by rabbiveesh** [comment]
I write in Perl. I would be happy to work out open sourcing the client library I wrote

**Comment by mendaxjerin** [comment]
Golang.

**Comment by appasahebnage** [comment]
Hey @ptaylor9999, I am interested in making a contribution to the Amazon Ads API using JavaScript/TypeScript.
